cache/lru_cache.cc

```diff
@@ -263,10 +297,24 @@ void LRUCacheShard::MaintainPoolSize() {
     lru_low_pri_ = lru_low_pri_->next;
     assert(lru_low_pri_ != &lru_);
     lru_low_pri_->SetInHighPriPool(false);
+    lru_low_pri_->SetInLowPriPool(true);
     size_t total_charge =
         lru_low_pri_->CalcTotalCharge(metadata_charge_policy_);
     assert(high_pri_pool_usage_ >= total_charge);
     high_pri_pool_usage_ -= total_charge;
+    low_pri_pool_usage_ += total_charge;
+  }
+
+  while (low_pri_pool_usage_ > low_pri_pool_capacity_) {
```

Comment:
I don't understand that why the actual link is not updated, but just update the usage_?

Reply:
Three priorities are composed in one link, like below
```
A---->B---->C---->D---->E---->F
      ^           ^           ^
      |           |           |
   bottom        low         high
```
So when one priority is full, only need to move the pointer of lower priority